packages/material-renderers/src/complex/MaterialEnumArrayRenderer.tsx

```diff
@@ -53,6 +89,7 @@ export const MaterialEnumArrayRenderer = ({
                 key={option.value}
                 control={
                   <MuiCheckbox
+                    id={id}
```

Comment:
I don't think we should add the id here. With this addition, all checkboxes of this multi enum would have the same id. I'd just remove this again. What do you think?

Reply:
This one is trickier.

I didn't add the id field, it was already there buried in otherProps.
I surfaced it only because I needed it for the htmlFor field of the FormLabel.

Also, it is required by MuiCheckBox:
![image](https://github.com/eclipsesource/jsonforms/assets/13748802/608ba7c9-0844-44eb-bf0c-00355aa30709)

What I propose is to concat the value as done for the key field:
```
  id={id+ '-checkbox-' + option.value}
  key={'checkbox-' + option.value}
```

But once you do that,
* The `htmlFor={id}` on `FormLabel` has no sense and should be removed
  * _And then_, it make no sense on MaterialRadioGroup where I got it to start with
* The field `id={option.value}` on `FormControlLabel` is also wrong (same id if the option is in another chekbox), and may be `id={id+ '-' + option.value}` or `id={id+ '-checkbox-label-' + option.value}`

Let me know what you think @lucas-koehler !

Reply:
> I didn't add the id field, it was already there buried in otherProps.

Ah I missed that, sorry.

I think your suggestion sounds good, to summarize:
- Remove `htmlFor={id}` from `FormLabel`
- Adapt ids of `FormControlLabel` from `id={option.value}` to `id={id+ '-label-' + option.value}` (I think just `label` instead of `checkbox-label` in combination with the value is enough)
- Adapt id on the checkbox to `id={id+ '-' + option.value}`

Reply:
Done !
[e32cbae](https://github.com/eclipsesource/jsonforms/pull/2297/commits/e32cbae1d5d389cc6c9249f044108e88a9049afa)